components/tag-list/test/tag-list-item-mixin-consumer.js

```diff
@@ -0,0 +1,73 @@
+import '../../dropdown/dropdown.js';
```

Comment:
This component is basically to help with development and is only used for the test and demo pages.  It's a stripped down version of what the `d2l-tag-list-item-user` will be, which will live in another repo.